cgroup2/manager_test.go

```diff
@@ -156,9 +156,10 @@ func TestKill(t *testing.T) {
 		if err := cmd.Start(); err != nil {
 			t.Fatal(err)
 		}
-		if cmd.Process == nil {
-			t.Fatal("Process is nil")
-		}
+		t.Cleanup(func() {
+			_ = cmd.Process.Kill()
```

Comment:
Maybe we should consider to add Pdeathsig to the process. Even if the test process has been killed and no chance to kill the sleep, the kernel will help us to do that.

REF: https://pkg.go.dev/syscall#SysProcAttr

Reply:
Not a bad idea, let me trial this tonight

Reply:
Swapped to this, thanks @fuweid